src/Spice86.Core/Emulator/OperatingSystem/Structures/DosDiskTransferArea.cs

```diff
@@ -105,4 +107,10 @@ public class DosDiskTransferArea : MemoryBasedDataStructure {
         get => GetZeroTerminatedString(FileNameOffset, FileNameSize);
         set => SetZeroTerminatedString(FileNameOffset, value, FileNameSize);
     }
+
+    /// <summary>
+    /// Gets or sets the result id that identifies the search results to use for the FindNext function.
+    /// This is not an official spec, but a Spice86 specific implementation.
+    /// </summary>
+    public ushort ResultId { get => UInt16[ResultIdOffset]; set => UInt16[ResultIdOffset] = value; }
```

Comment:
I could not find a reference to this in the code base. Should not it be referenced in the DosFileManager ?

Reply:
Ah yes, this is used by my C# implementation of the [StdIO C library](https://github.com/JorisVanEijden/ReverseBak/blob/f3c6fb5bb86c8bff84ccfabc1a136f79e331965f/BetrayalAtKrondor/Overrides/Libraries/StdIO.cs#L292) used by Krondor. 

You introduced the "SearchId" later to fix it in spice. But that is marked `internal` so my library can't use it.
I'll just copy this version to my library and leave it alone in Spice86.

Reply:
It can be made public.

Reply:
Nah, it's fine. I moved it to my own code.